src/shared/sam/localLambdaRunner.ts

```diff
@@ -249,10 +198,11 @@ export async function invokeLambdaFunction(
 
     const eventPath: string = path.join(config.baseBuildDir!!, 'event.json')
     const environmentVariablePath = path.join(config.baseBuildDir!!, 'env-vars.json')
+    const env = JSON.stringify(getEnvironmentVariables(config.lambda?.environmentVariables ?? {}))
```

Comment:
Make `getEnvironmentVariables` handle the parameter as optional and remove this logic. You're already implying it in that function based on the ternary operator there.